x/evm/keeper/migrations.go

```diff
@@ -21,3 +22,8 @@ func NewMigrator(keeper Keeper) Migrator {
 func (m Migrator) Migrate1to2(ctx sdk.Context) error {
 	return v2.MigrateStore(ctx, &m.keeper.paramSpace)
 }
+
+// Migrate2to3 migrates the store from consensus version v2 to v3
```

Comment:
`// Migrate2to3 migrates the store from consensus version v2 to v3`
Is the comment misleading? why it says **consensus version**?

Reply:
Module's version for state migration for consensus state machine.